common/requirements.txt

```diff
@@ -4,3 +4,4 @@ requests==2.22.0
 PyMySQL==0.10.1
 boto3==1.15.9
 ipfsapi==0.4.3
```

Comment:
we can remove `ipfsapi` from `common/requirements.txt`  since we are using different package now